src/libopensc/pkcs15-itacns.c

```diff
@@ -30,17 +30,25 @@
 #include <config.h>
 #endif
 
-#include "pkcs15.h"
-#include "log.h"
-#include "cards.h"
-#include "itacns.h"
 #include <stdlib.h>
 #include <string.h>
 #include <stdio.h>
+
+#ifdef ENABLE_OPENSSL
+#include <openssl/x509.h>
+#include <openssl/x509v3.h>
+#endif
+
+#include "libopensc/internal.h"
+#include "libopensc/pkcs15.h"
+#include "libopensc/log.h"
+#include "libopensc/cards.h"
+#include "libopensc/itacns.h"
```

Comment:
They were move to be after libopensc/internal.h

libopensc/internal.h includes the libopensc/sc-ossl-compat.c 

libopensc/sc-ossl-compat.c  includes openssl/opensslv.h and openssl/opensslconf.h
which should be the first openssl header files included. 

the names now include "libopensc/" as is done in other source files. 

This makes the order of the includes more inline with other source files.


and "libopensc/" was add to each to match what most other source files do.